### PR TITLE
Bulma upgrade & minor bug fixing (pt 2)

### PR DESCRIPTION
**Pull request by tejashah88**
Leading off from #20 due to accidental early merge

Summary of changes:
* Upgraded UI look-and-feel to use Bulma
* Added home page and sensor dashboard
* Replaced custom hamburger with Bulma's QuickView
* Bug fixes

### CODE REVIEW

File: webapp/templates/layout.html [hunk 1]
```diff
@@ -10,8 +10,6 @@
         <link rel="shortcut icon" href="static/raspberry-pi-robot.ico" />
     </head>
     <body class="has-background-black-ter">
-        <script src="/static/scripts/hamburger.js"></script>
```
Comment (2bndy5) [inline webapp/templates/layout.html:13]: why did you take out the hamburger button/script? I was going to replace Bulma's awkward nav menu with the sidebar.
Note: Bulma's hamburger button mandated to live on the right side of the nav bar, while the side bar that the hamburger button opens up is stashed/animated from the left (like literally every app I know)

Comment (2bndy5) [reply webapp/templates/layout.html:13]: we can force Bulma not to generate their hamburger button in favor for ours

Comment (tejashah88) [reply webapp/templates/layout.html:13]: I was going to integrate the new nav bar first, and then add the hamburger back once the upgrade is mostly complete

Comment (tejashah88) [reply webapp/templates/layout.html:13]: I found a way for Bulma's QuickView to be left-justified! It's not documented but it does exist as a CSS class called `is-left`.

Link: https://github.com/Wikiki/bulma-quickview/blob/master/src/sass/index.sass#L38.

**Pull request merged**